napalm_fortios/fortios.py

```diff
@@ -343,6 +343,54 @@ def get_interfaces_counters(self):
     def get_lldp_neighbors(self):
         return {}
 
+    def get_firewall_policies(self):
+        cmd = self.execute_command_with_vdom('show firewall policy')
+        policy = dict()
+        policy_id = None
+        default_policy = dict()
+        position = 0
+        for line in cmd:
+            policy_data = re.sub("\s+", " ", line).lstrip()
+            if policy_data.find("edit") == 0:
+                policy_id = policy_data.split()[1]
+                policy[policy_id] = dict()
+            if policy_id is not None:
+                if len(policy_data.split()) > 2:
+                    policy_setting = policy_data.split()[1]
+                    policy[policy_id][policy_setting] = policy_data.split()[2].replace("\"", "")
+        for key in policy:
+            if 'status' in  policy[key]:
```

Comment:
The whole key does not exist when the policy is disabled. That's why I only can check if the key is there.